astropy/visualization/scripts/fits2bitmap.py

```diff
@@ -182,6 +186,18 @@ def main(args=None):
             "(Default is 0.1)."
         ),
     )
+    parser.add_argument(
```

Comment:
maybe we could use [mutually exclusive groups](https://docs.python.org/3/library/argparse.html#mutual-exclusion) here to forbid that the new arguments be passes *alongside* their deprecated counterparts.

Reply:
Good idea!  I've updated this PR to use mutually exclusive groups.